### PR TITLE
applovin/mrec_zone_and_open_bidding_watermark_support

### PR DESCRIPTION
**Pull request by Vedant-Mehta39**
1. Adds support for Medium Rectangle Ads 
2. Adds support for watermarks when bidding

### CODE REVIEW

**Review by thomasmso — APPROVED** [review]

File: ThirdPartyAdapters/applovin/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 #### Version 11.8.2.0 (In progress)
 - Verified compatibility with AppLovin SDK 11.8.2.
+- Added support for watermark when bidding.
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/applovin/CHANGELOG.md:5]: Nit, `- Added watermark support for bidding ads.`

File: ThirdPartyAdapters/applovin/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
+- Added support for Medium Rectangle(MREC) ads.
```
Comment (joshh-devrel) [inline ThirdPartyAdapters/applovin/CHANGELOG.md:6]: Nit, `- Added support for MREC size banner ads.`

**Comment by Vedant-Mehta39** [comment]
Changes Made

**Comment by Vedant-Mehta39** [comment]
Changes Made:

Deleted `AppLovinRtbBannerRenderer` and removed it's usages

**Comment by joshh-devrel** [comment]
@Vedant-Mehta39 Can you rebase this PR? Thanks!

**Comment by Vedant-Mehta39** [comment]
Changes Made

Branch rebased

**Pull request merged**